selfdrive/car/card.py

```diff
@@ -113,6 +115,10 @@ def __init__(self, CI=None, RI=None) -> None:
     if not disengage_on_accelerator:
       self.CP.alternativeExperience |= ALTERNATIVE_EXPERIENCE.DISABLE_DISENGAGE_ON_GAS
 
+    # mads
+    MadsParams().set_alternative_experience(self.CP)
+    MadsParams().set_car_specific_params(self.CP)
```

Comment:
Would it be possible to maybe add these as part of the ctor of `MadsParams()`? would that make sense?

Reply:
something like 
```py
MadsParams(self.CP)
```

Reply:
While doable, `set_alternative_experience` and `set_car_specific_params` are both static methods meant to be used independently of any instance state. We don't necessarily need to instantiating the class for these since we don't need to create a new object.

Reply:
`MadsParams().set_car_params(self.CP)` ?